app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_with model: comment, url: new_comment_path,  method: :comment, local: true, scope: :comment do |f| %>
+  <%= f.hidden_field :author_id, value: params[:id] %><br>
+  <%= f.hidden_field :post_id, value: params[:post_id] %><br>
+
+  <div class="flex-row col-flex">
+    <p>
+      <%= f.text_area :text, size: "50x12",
+      placeholder: "Write your post here...",
+      value: ""
+        %>
+    </p>
+    <p>
+      <%= f.submit %>
+    </p>
+  </div>
+<% end %>
```

Comment:
- [ ] Comment form should have its route as I mentioned below `/users/:user_id/posts/:post_id(.:format)                                                          comments/new` you can add a button like `add a comment` maybe that lead you to the comment form

Reply:
Addressed ✔